doc/REST-interface.md

```diff
@@ -84,6 +84,12 @@ Returns various state info regarding block chain processing.
 Only supports JSON as output format.
 Refer to the `getblockchaininfo` RPC help for details.
 
+#### Deployment info
+`GET /rest/deploymentinfo/<BLOCKHASH>.json`
```

Comment:
Maybe it's better to specify that the `<BLOCKHASH>` parameter is optional and `rest/deploymentinfo.json` will use the hash of current chain tip.

Reply:
Nice!